ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1166 @@
+        const chart = this.chart,
+            currentBreadcrumb = this.list[this.level - 1];
+
+        if (chart.drillUpButton) {
+            chart.drillUpButton.attr({
+                text: this.getButtonText(currentBreadcrumb)
+            });
+        }
+    }
+
+    /**
+     * Destroy the chosen breadcrumbs group
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#destroy
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public destroy(this: Breadcrumbs): void {
+
+        this.destroySingleButton();
+
+        // Destroy elements one by one. It's necessary beacause
+        // g().destroy() does not remove added HTML
+        this.destroyListElements(true);
+
+        // Clear the breadcrums list array.
+        if (this.group) {
+            this.group.destroy();
+        }
+        this.group = void 0 as any;
+
+    }
+    /**
+     * Destroy the elements' buttons and separators.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#destroyListElements
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public destroyListElements(
+        this: Breadcrumbs,
+        force?: boolean
+    ): void {
+        const elementList = this.elementList;
+
+        objectEach(elementList, (element, level): void => {
+            if (
+                force ||
+                !elementList[level].updated
+            ) {
+                element = elementList[level];
+                element.button && element.button.destroy();
+                element.separator && element.separator.destroy();
+                delete element.button;
+                delete element.separator;
+                delete elementList[level];
+            }
+        });
+
+        if (force) {
+            this.elementList = {};
+        }
+    }
+
+    /**
+     * Destroy the single button if exists.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#destroySingleButton
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public destroySingleButton(): void {
+        if (this.chart.drillUpButton) {
+            this.chart.drillUpButton.destroy();
+            this.chart.drillUpButton = void 0;
+        }
+    }
+    /**
+     * Reset state for all buttons in elementList.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#resetElementListState
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public resetElementListState(
+        this: Breadcrumbs
+    ): void {
+        objectEach(
+            this.elementList,
+            (element): void => {
+                element.updated = false;
+            }
+        );
+    }
+
+    /**
+     * Reset state for all buttons in elementList.
```

Comment:
Incorrect description. Seems to be description from `resetElementListState` method